README.md

```diff
@@ -0,0 +1,255 @@
+
+- [Node JS](https://nodejs.org/en/)
+
+Follow the link below to install NPM
+
+- [NPM](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+
+Follow the link below to install Yarn
+
+- [Yarn](https://classic.yarnpkg.com/en/docs/install/#windows-stable)
+
+Once you have installed Node JS and NPM or Yarn, install the following packages globally:
+
+```sh
+  npm install -g stylelint
+  npm install -g eslint
+  npm install react-icons
+```
+
+OR if on Yarn,
+
+```sh
+  yarn global add stylelint
+  yarn global add eslint
+  yarn add react-icons
+```
+
+However, whilst everything is included in the package.json file, you can simply run:
+
+```sh
+  npm install
+```
+
+OR if on Yarn,
+
+```sh
+  yarn install
+```
+
+Don't know how to use Package Managers? Check out the [Beginner's Guide](https://www.sitepoint.com/yarn-vs-npm/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/10-menachi/anime-facts.git
+```
+
+### Usage
+
+To run the project:
+
+```sh
+  cd anime-facts
+  npm run start
+```
+
+OR if on Yarn,
+
+```sh
+  cd anime-facts
+  yarn start
+```
+
+The website is hosted on localhost:5173 by default. <br />
+
+To change the port to the port of your choice, run:
+
+```sh
+  npm run start --port=YOUR_PORT
+```
+
+Wanna learn more about React? Check out the [React Documentation](https://reactjs.org/docs/getting-started.html)
+Wanna learn more about Vite? Check out the [Vite Documentation](https://vitejs.dev/guide/)
+
+### Run tests
+
+To run the tests:
+
+```sh
+  npm run test
+```
+
+OR if on Yarn,
+
+```sh
+  yarn test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Christian Wamalwa**
+
+- GitHub: [@githubhandle](https://github.com/10-menachi)
+- Twitter: [@twitterhandle](https://twitter.com/christian_timbe)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/chris-droid)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **User Authentication**
+- [ ] **Edit Anime**
+- [ ] **Update Anime Watching Progress**
+- [ ] **View Anime comments**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give it a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me this opportunity to learn and build this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
##### **Creative Commons attribution**

- [ ] Please ensure that the "Acknowledgements" section of your README gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-\(Free-PSDs\)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). :100: